lib/robot.js

```diff
@@ -25,6 +25,25 @@ class Robot {
     });
   }
 
+  addEventToListeners(event, callback) {
```

Comment:
From what I can tell it needs to be a class method because of `this` unless we decide to take some sort of different approach altogether.

Does JS have a way to make this a private method? Even if not, I don't think it matters too much. This function basically does the same as robot.on() did before.